packages/sanity/package.config.ts

```diff
@@ -4,6 +4,12 @@ import {defineConfig} from '@sanity/pkg-utils'
 export default defineConfig({
   ...baseConfig,
 
+  define: {
+    ...baseConfig.define,
+    PRESENTATION_ENABLE_LIVE_DRAFT_EVENTS:
+      process.env.PRESENTATION_ENABLE_LIVE_DRAFT_EVENTS === 'true',
```

Comment:
This is used to feature flag a new approach to drive live previews, which uses Sanity Live Content API for draft content (which currently requires vX, which is why it's off by default)